vaadin-upload.html

```diff
@@ -0,0 +1,225 @@
+      }
+
+      [part="drop-label-icon"] {
+        display: inline-block;
+        margin-right: 8px;
+      }
+
+      [part="drop-label-icon"]::before {
+        content: var(--material-icons-upload);
+        font-family: material-icons;
+        font-size: var(--material-icon-font-size);
+        line-height: 1;
+      }
+
+      /* Ripple */
+
+      :host::before {
+        content: "";
+        position: absolute;
+        width: 100px;
+        height: 100px;
+        border-radius: 50%;
+        top: 50%;
+        left: 50%;
+        pointer-events: none;
+        background-color: var(--material-primary-color);
+        opacity: 0;
+        transform: translate(-50%, -50%) scale(0);
+        transition: transform 0s cubic-bezier(0.05, 0.8, 0.5, 1), opacity 0.4s linear;
+        transition-delay: 0.4s, 0s;
+      }
+
+      :host([dragover-valid])::before {
+        transform: translate(-50%, -50%) scale(6);
+        opacity: 0.1;
+        transition-duration: 2s, 0.1s;
+        transition-delay: 0s, 0s;
+      }
+    </style>
+  </template>
+</dom-module>
+
+<dom-module id="material-upload-file" theme-for="vaadin-upload-file">
+  <template>
+    <style>
+      :host {
+        padding: 8px 0;
+      }
+
+      :host(:not(:first-child)) {
+        border-top: 1px solid var(--material-divider-color);
+      }
+
+      [part="row"] {
+        display: flex;
+        align-items: flex-start;
+        justify-content: space-between;
+      }
+
+      [part="status"],
+      [part="error"] {
+        color: var(--material-secondary-text-color);
+        font-size: var(--material-caption-font-size);
+      }
+
+      [part="info"] {
+        display: flex;
+        align-items: baseline;
+        flex: auto;
+      }
+
+      [part="meta"] {
+        width: 0.001px;
+        flex: 1 1 auto;
+      }
+
+      [part="name"] {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        font-size: var(--material-small-font-size);
+      }
+
+      [part="commands"] {
+        display: flex;
+        flex: none;
+        font-family: material-icons;
+        font-size: var(--material-icon-font-size);
+      }
+
+      [part="done-icon"],
+      [part="warning-icon"] {
+        padding: 4px;
+        margin: -4px 4px -4px 0;
+        font-family: material-icons;
+        font-size: var(--material-icon-font-size);
+      }
+
+      /* When both icons are hidden, let us keep space for one */
+      [part="done-icon"][hidden] + [part="warning-icon"][hidden] {
+        display: block !important;
+        visibility: hidden;
+      }
+
+      [part="done-icon"],
+      [part="warning-icon"] {
+        font-size: var(--material-icon-font-size);
+        line-height: 1;
+      }
+
+      [part="start-button"],
+      [part="retry-button"],
+      [part="clear-button"] {
+        flex: none;
+        color: var(--material-secondary-text-color);
+        line-height: 1;
+        padding: 4px;
```

Comment:
Quite small touch target size. Increasing padding up to 8px on each side does not seem to harm much here.

Not a blocker though, we can adjust this later.

Reply:
Increased the size (with 8px padding).